app/models/feed.rb

```diff
@@ -93,19 +105,27 @@ def default_runtime_settings?
 
   def published_url(include_token = nil)
     if private?
-      private_path = "#{podcast.base_private_url}/#{published_path}"
-
-      if include_token == true
-        "#{private_path}?auth=#{tokens.first&.token}"
-      elsif include_token.present?
-        "#{private_path}?auth=#{include_token}"
-      elsif include_token.nil?
-        "#{private_path}{?auth}"
-      else
-        private_path
-      end
+      published_private_url(include_token)
+    else
+      published_public_url
+    end
+  end
```

Comment:
Abstacted the logic into 2 separate methods - so in the view, I can render them both and `d-none` one of them.

This allows us to show the correct `Private Feed URL` as you check/uncheck the "Private" checkbox.